docs/sources/dev/schemas.md

```diff
@@ -1 +1,41 @@
 # Creating or Modifying a Schema
+
+The matching tool can be extended to work with schemas not defined in the initial release. This document defines how this can be done and what limitations are involved.
+
+## Modifying an Existing Schema
+
+To modify the columns or checks in an existing schema:
+
+1. Find the schema in [webapp/schemas/uploader/](https://github.com/dssg/matching-tool/tree/master/webapp/schemas/uploader).
+
+2. Modify the name, datatype, or constraints for an individual field, or the 'primaryKey' that should uniquely define each row. If it is not clear how to modify an individual schema to, for instance, express a constraint in a given way, refer to either the [goodtables-py docs](https://github.com/frictionlessdata/goodtables-py) (the general library used by the matching tool) or the [webapp.validations](https://github.com/dssg/matching-tool/blob/master/webapp/webapp/validations/__init__.py) module (the file where custom checks for the matching tool are defined).
+
+3. If any of the column names, data types, or primary key have changed, you will have to delete the installation's master table as this information is encoded in the master table's schema. Refer your administrator to [Clearing a Master Table for a Schema](../admin/updating.md#clearing-a-master-table-for-a-schema).
+
+
+## Creating a new Schema
+
+To create a new schema:
+
+1. Create a JSON file containing the schema in [webapp/schemas/uploader/](https://github.com/dssg/matching-tool/tree/master/webapp/schemas/uploader), obeying the same name scheme (`lowercase-separated-by-dashes.json`). Copying an example from one of the existing schemas is recommended. The important parts of the schema:
+    - The top-level `name` field simply contains a human-readable name for the schema. This is used in the web UI.
+    - The `fields` field contains a list of column names, their datatypes, and any necessary constraints on them. If you have questions about standard constraints that are not clear from the existing .json files, refer to the [goodtables-py docs](https://github.com/frictionlessdata/goodtables-py) (the general library used by the matching tool). When possible, use the same field names and types for columns that are present in existing schemas, as this will minimize the amount of matcher re-configuration needed in step 4.
```

Comment:
I would add "constraints" to the list of things that ought to be common across similarly named fields